tests/src/e2e.rs

```diff
@@ -488,21 +491,38 @@ mod tests {
             .exp_string("Started node")
             .map_err(|e| eyre!(format!("{}", e)))?;
 
-        // 2. Submit a an invalid transaction (amount > source's balance)
+        // 2. Submit a an invalid transaction (trying to mint tokens should fail
```

Comment:
I had to change the invalid tx example here for https://github.com/anoma/anoma/pull/419/files#r703358474 change, because now the overflow that's caused by having `amount > source's balance` doesn't fail the validation, but instead crashes the tx before we run VPs, which is preferable, because the VP would fail anyway on overflow/under-flow. I've added a new example tx in `wasm`_for_tests`:
```rust
/// A tx that attempts to mint tokens in the transfer's target without debiting
/// the tokens from the source. This tx is expected to be rejected by the
/// token's VP.
```